### PR TITLE
Add poblacionData and fun get data poblacion

### LINKED ISSUE #2: Fix Grammar Errors

### LINKED ISSUE #3: Add population

### PR DESCRIPTION
**Pull request by UdanPrg**
Solves #2 #3

### CODE REVIEW

File: Senso_2018_pobación_por_departamentos.jpg
Comment (COANelmejor) [inline Senso_2018_pobación_por_departamentos.jpg:1]: How and where is this image used? 🤔

File: data.js [hunk 1]
```diff
@@ -776,21 +793,78 @@ const municipiosObj = {
...
+  "Quiché": "949,261",
+  "Retalhuleu": "326,828",
+  "Sacatepéquez": "330,469",
+  "San Marcos": "1,032,277",
+  "Santa Rosa": "396,607",
+  "Sololá": "421,583",
+  "Suchitepéquez": "554,695",
+  "Totonicapán": "418,569",
+  "Zacapa": "245,374",
+}
```
Comment (COANelmejor) [inline data.js]: Convert the numbers to "Number" type
It is more consistent for programming
User underscore (_) to separate the thousands if you want.
``` js
// Example
{
  "Sacatepéquez": 330469,
  "San Marcos": 1_032_277,
}
  
 ```

File: index.js [hunk 2]
```diff
@@ -22,6 +22,15 @@ function municipios(departamento){
...
 
+function poblacion(departamento){
+  if (departamento == null) {
+    return data.poblacionDeptoObj;
+  } else {
+    const ldep = repDep(departamento);
+    return data.poblacionDepto[ldep];
+  }
+}
+
```
Comment (COANelmejor) [inline index.js]: Add a jsDoc Comment to the function.
So developers can check what the function does.
``` js
/** # Title
 *  info and example
 */
function poblacion(departamento){
  if (departamento == null) {
    return data.poblacionDeptoObj;
  } else {
    const ldep = repDep(departamento);
    return data.poblacionDepto[ldep];
  }
}
```

**Review by COANelmejor — CHANGES_REQUESTED** [review]
Thanks for fixing the grammar errors.
Review the changes requested, please.
Also, add the info on new changes and examples of how to use into the `Readme.md` file.
And add a Description to your PR.

File: Senso_2018_pobación_por_departamentos.jpg
Comment (UdanPrg) [reply Senso_2018_pobación_por_departamentos.jpg:1]: Reference and source.

I am also going to add an .xlsx file that contains the departments and municipalities
And I will also add to the readme.md the sources of the senso 2018 of Guatemala

File: data.js (hunk 1 above)
Comment (UdanPrg) [reply data.js]: Always as string data?

File: index.js (hunk 2 above)
Comment (UdanPrg) [reply index.js]: Done

File: data.js (hunk 1 above)
Comment (UdanPrg) [reply data.js]: Done

File: index.js (hunk 2 above)
Comment (UdanPrg) [reply index.js]: Solve

File: Senso_2018_pobación_por_departamentos.jpg
Comment (COANelmejor) [reply Senso_2018_pobación_por_departamentos.jpg:1]: Info and files unrelated to the code should not be in the repo unless it is needed to use in the code.

File: data.js [hunk 3]
```diff
@@ -776,21 +793,78 @@ const municipiosObj = {
...
+  "Quiché": "949_261",
+  "Retalhuleu": "326_828",
+  "Sacatepéquez": "330_469",
+  "San Marcos": "1_032_277",
+  "Santa Rosa": "396_607",
+  "Sololá": "421_583",
+  "Suchitepéquez": "554_695",
+  "Totonicapán": "418_569",
+  "Zacapa": "245_374",
+}
```
Comment (COANelmejor) [inline data.js]: The numbers of keys still are "Strings." They should be "Number."

**Review by COANelmejor — CHANGES_REQUESTED** [review]
Change the number "strings" to numbers.

File: data.js (hunk 3 above)
Comment (UdanPrg) [reply data.js]: Solve

**Review by COANelmejor — APPROVED** [review]
Ready to merge

**Review by COANelmejor — CHANGES_REQUESTED** [review]
Add examples of use of new methods on Readme.md

**Review by COANelmejor — APPROVED** [review]

**Pull request merged**